ui/css/itcss/components/newui-sections.scss

```diff
@@ -101,14 +85,6 @@
   }
 }
 
-.fiat-amount {
-  text-transform: uppercase;
-}
```

Comment:
Not used

![Screenshot 2025-02-07 at 11 55 15 AM](https://github.com/user-attachments/assets/61f2c079-5027-42f1-8ee4-2d9c674603f4)